### ISSUE TITLE
Syntax error in 0.4.0

### ISSUE THREAD

**Issue by maxlapides** [kiwi-bop/intl_generator#20]
Hi there! Thank you for releasing version 0.4.0!

I am now getting this error:

```
../../.pub-cache/hosted/pub.dartlang.org/intl_generator-0.4.0/lib/extract_messages.dart:169:74: Error: Too many positional arguments: 0 allowed, but 5 found.
Try removing the extra positional arguments.
  final FormalParameterList _emptyParameterList = FormalParameterListImpl(
                                                                         ^
../../.pub-cache/hosted/pub.dartlang.org/analyzer-5.2.0/lib/src/dart/ast/ast.dart:5342:3: Context: Found this candidate, but the arguments don't match.
  FormalParameterListImpl({
  ^^^^^^^^^^^^^^^^^^^^^^^
pub finished with exit code 1
```

I think we probably need to revert the changes in this commit: https://github.com/kiwi-bop/intl_generator/commit/64c33374b2daa0fa8d3cb68ab1cdec9e84b31123. What do you think @jaumard?

**Comment by maxlapides** [comment]
I think this issue is also being raised by the pub.dev scores: https://pub.dev/packages/intl_generator/score

**Comment by jaumard** [comment]
Fixed in 4.0.1 sorry for that
